letsencrypt-auto

```diff
@@ -61,10 +58,34 @@ if test "`id -u`" -ne "0" ; then
       done
       su root -c "$args"
     }
-    SUDO=su_sudo
-  fi
+}
+
+if /usr/bin/env | grep '^SUDO=' 1>/dev/null 2>&1; then
+  case "$SUDO" in
```

Comment:
For consistency with the other env vars letsencrypt-auto checks, can we rename this `LE_AUTO_SUDO`? (Also, I share @pde's fear of an accidental collision with some other $SUDO hanging around.)